docs/transport.asciidoc

```diff
@@ -7,34 +7,60 @@ When available, it handles connecting to multiple nodes in the cluster, rotating
 
 It does not handle calling the {es} or Enterprise Search APIs.
 
-For optimal performance, use a HTTP library which supports persistent ("keep-alive") connections, such as https://github.com/toland/patron[patron] or https://github.com/typhoeus/typhoeus[Typhoeus]. Require the library (require 'patron') in your code, and it will be automatically used.
+This library uses https://github.com/lostisland/faraday[Faraday] by default as the HTTP transport implementation. We test it it with Faraday versions 1.x and Faraday 2.x.
+
+For optimal performance, use a HTTP library which supports persistent ("keep-alive") connections, such as https://github.com/toland/patron[patron] or https://github.com/typhoeus/typhoeus[Typhoeus]. Require the library (`require 'patron'`) in your code for Faraday 1.x or the adapter (`require 'faraday/patron'`) for Faraday 2.x, and it will be automatically used.
+
+Currently these libraries are supported:
+
+* https://github.com/toland/patron[Patron]
+* https://github.com/typhoeus/typhoeus[Typhoeus]
+* https://rubygems.org/gems/httpclient[HTTPClient]
+* https://rubygems.org/gems/net-http-persistent[Net::HTTP::Persistent]
+
+**Note on https://github.com/typhoeus/typhoeus[Typhoeus]**: You need to use v1.4.0 or up since older versions are not compatible with Faraday 1.0.
```

Comment:
NOTE is an admonition, so the statement will be a bit more prominent.

```suggestion
NOTE: Use https://github.com/typhoeus/typhoeus[Typhoeus] v1.4.0 or up since older versions are not compatible with Faraday 1.0.
```